entryDetail.go

```diff
@@ -516,3 +490,13 @@ func (ed *EntryDetail) CreditOrDebit() string {
 	}
 	return ""
 }
+
+// GetAddenda05 returns a slice of Addenda05 for the batch
+func (ed *EntryDetail) GetAddenda05() []*Addenda05 {
```

Comment:
This method is never called in the PR, do we need it?

Reply:
Not right now, I'll remove it, if we need it later we can add it.